core/domain/exp_services.py

```diff
@@ -70,6 +70,9 @@
 # search query.
 MAX_ITERATIONS = 10
 
+# Asset directories that should not be bundled for download.
+DOWNLOAD_RESTRICTED_ASSET_DIRS = ['audio']
```

Comment:
Should we scope this to the function level? Currently, it seems like we restrict downloading audio files.

Reply:
Per discussion with @DubeySandeep I have moved the constant inside the function and made it lowercase.